Network/HTTP/Req.hs

```diff
@@ -1637,6 +1644,8 @@ class HttpResponse response where
     -> IO response
        -- ^ The final result
 
+  acceptHeader :: forall scheme. Option scheme
```

Comment:
Perhaps need to add a comment to this. 
Maybe describe that this is tacked on to the end of the headers so the user can override it if he wants to.